docker-compose.yml

```
@@ -200,14 +86,15 @@ services:
       - --datadir=/execution/erigon
       - --nat=extip:10.100.100.13
       - --db.size.limit=1GB
+      - --networkid=32382
     ports:
       - 18551:8551
       - 18545:8545
       - 18546:8546
```

Now that there's no `geth` , I think we can drop the `1` prefix here.
```suggestion
      - 8551:8551
      - 8545:8545
      - 8546:8546
```

---

good catch!